api.swagger.yaml

```diff
@@ -2846,7 +2846,15 @@ components:
                 - $ref: '#/components/schemas/title-or-uuid-writable-list-schema'
     user-schema-create:
       allOf:
-        - $ref: '#/components/schemas/user-schema-update'
+        - $ref: '#/components/schemas/user-schema'
+        - properties:
+            job_roles:
+              allOf:
+                - $ref: '#/components/schemas/title-or-uuid-writable-list-schema'
```

Comment:
I don't understand the reasoning behind this change?

Reply:
I had to rejig the schemas to allow this change and still allow job roles to be specified by name on create/update also from the sierra club CR